TestCases/compliance-level-3/0083-feel-unicode/0083-feel-unicode.dmn

```diff
@@ -9,6 +9,21 @@
         </literalExpression>
     </decision>
 
+    <decision name="decision_001_a" id="_decision_001_a">
+        <variable name="decision_001_a"/>
+        <literalExpression>
+            <text>string length("\U01F40E")</text>
+        </literalExpression>
+    </decision>
+
+    <decision name="decision_001_b" id="_decision_001_b">
+        <variable name="decision_001_b"/>
+        <literalExpression>
+            <!-- NOTE: DMN 1.3 spec says this should be a "ὀ" -->
```

Comment:
```suggestion
            <!-- NOTE: the published DMN 1.3 spec has a typo reporting "ὀ" instead of the emoji, which has been redacted with https://issues.omg.org/browse/DMN14-72 -->
```

Reply:
Thanks @tarilabs - appreciated.  I have pushed your requested change but updated the given issue URL.  That URL you've supplied there requires a login.  The https://issues.omg.org/issues/DMN14-72 is public and does not.

btw, nice to know I wasn't crazy with that wrong character in the spec.  I really did look at that one for some time thinking I must be wrong, but ended up figuring I'd just go with what I thought was correct.  I later saw the annotation in the DMN 1.3 PDF changelog saying "note: special character may be wrong".